src/widgets/ErrorDialog.h

```diff
@@ -46,11 +46,15 @@ class HtmlTextHelpDialog : public BrowserFrame
 public:
    HtmlTextHelpDialog() : BrowserFrame()
    {
+#if !wxCHECK_VERSION(3, 1, 0)
```

Comment:
On 5/16/2015 3:21 PM, Leland Lucius wrote:
>
> In src/widgets/ErrorDialog.h 
> <https://github.com/audacity/audacity/pull/44#discussion_r30464886>:
>
> > @@ -46,11 +46,15 @@ class HtmlTextHelpDialog : public BrowserFrame
> >  public:
> >     HtmlTextHelpDialog() : BrowserFrame()
> >     {
> > +#if !wxCHECK_VERSION(3, 1, 0)
>
> What happens if we just bypass MakeModal this way? Does it leave the 
> dialog modeless? Will that allow unwanted interaction with the project 
> window?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/audacity/audacity/pull/44/files#r30464886>.
>
MakeModal no longer exists in wx31

this is the hack i said i didn't like when submitting the earlier wx3 
patches but i haven't figured out a better solution

dave

-- 
David A. Avery
Avery Design
  Technical Theatre
    Lighting Design
    Scenery Construction
    Sound Design
    Live Recording
   (303) 725-7515 (cell)